components/expenses/ExpenseFormPayeeStep.js

```diff
@@ -88,12 +87,20 @@ const getPayoutMethodsFromPayee = payee => {
   }
 
   // If the Payee is in the "Collective" family (Collective, Fund, Event, Project)
-  // Then the Account Balance should be its only option
+  // But not the Host itself (Self Hosted)
+  // Then we should add BANK_ACCOUNT and PAYPAL of the Host as an option
   if (payee && AccountTypesWithHost.includes(payee.type) && payee.id !== payee.host?.id) {
-    filteredPms = filteredPms.filter(pm => pm.type === PayoutMethodType.ACCOUNT_BALANCE);
+    if (!payoutMethods.find(pm => pm.type === PayoutMethodType.BANK_ACCOUNT)) {
+      const hostPayoutMethods = get(payee, 'host.payoutMethods') || EMPTY_ARRAY;
+      const hostBankAccountPayoutMethods = hostPayoutMethods
```

Comment:
Do we really want PayPal or should we start with Bank Transfer?

If, PayPal is ok, rename variable.